cmake/openenclave-config.cmake.in

```diff
@@ -45,25 +45,25 @@ endif ()
 include(CMakeFindDependencyMacro)
 find_dependency(Threads)
 if (UNIX)
-  if (NOT TARGET crypto)
+  if (NOT TARGET openenclave::import::crypto)
```

Comment:
Is `openenclave::import::` consistent with the rest of our imported libraries? I believe those brought in properly are imported to just the `openenclave::` namespace, which is fine as it avoids conflicts with anyone bringing in `openenclave` or `dl` or `crypto`. Since we don't re-export these imported libraries, the extra `import::` is unnecessary and, I think, inconsistent.

Reply:
Just update the patch, please take a look.

Reply:
Looks good.